### PR TITLE
feat: new music drop create flow

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**

### CODE REVIEW

File: packages/app/components/drop/create-drop-steps/create-drop-steps.tsx [hunk 1]
```diff
@@ -0,0 +1,1179 @@
+import { useState, useRef, useEffect, useMemo, useCallback } from "react";
```
Comment (intergalacticspacehighway) [inline packages/app/components/drop/create-drop-steps/create-drop-steps.tsx:1]: keeping every step in this file. We can split into different but i think it's fine. We need some pieces for paid drops. I'll restructure once we start working on paid drops.

File: packages/app/components/drop/create-drop-steps/music-drop-form-utils.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    getValues,
+    trigger,
+  } = useForm<any>({
+    resolver: yupResolver(dropValidationSchema as any),
+    reValidateMode: "onChange",
+    mode: "onBlur",
+    shouldFocusError: true,
+  });
+
+  console.log("form state ", errors);
```
Comment (alantoa) [inline packages/app/components/drop/create-drop-steps/music-drop-form-utils.tsx]: Is this log necessary?

Comment (intergalacticspacehighway) [reply packages/app/components/drop/create-drop-steps/music-drop-form-utils.tsx]: fixed!

**Review by alantoa — APPROVED** [review]
LGTM 🔥

**Pull request merged**